docs/kyma-environment-broker/03-08-kyma-versions.md

```diff
@@ -13,10 +17,11 @@ metadata:
   name: kyma-versions
   namespace: "kcp-system"
 data:
-  3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  GA_3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  SA_df29c526-0c3d-4e2c-ba41-8b69cde41961: "PR-3721"
```

Comment:
Consider adding an explanation that the value might be the version, the PR number, or `master`.